packages/fluentui/accessibility/src/behaviors/Tree/treeAsListBehavior.ts

```diff
@@ -6,14 +6,18 @@ import { treeItemAsListItemBehavior } from './treeItemAsListItemBehavior';
 
 /**
  * @specification
- * Adds role 'list' to 'root' slot.
+ * Adds attribute 'role=listbox' to 'root' slot if 'selectable' property is true. Sets the attribute to 'list' otherwise.
+ * Adds attribute 'aria-multiselectable=true' to 'root' slot if 'selectable' property is true. Does not set the attribute otherwise.
  */
 export const treeAsListBehavior: Accessibility<TreeBehaviorProps> = props => {
   const behavior = treeBehavior(props);
   return _.merge(behavior, {
     attributes: {
       root: {
-        role: 'list',
+        role: props.selectable ? 'listbox' : 'list',
+        ...(props.selectable && {
+          'aria-multiselectable': true,
```

Comment:
I think this might be missing in the `treeBehavior` as well. Should we added @kolaps33 ?

Reply:
I'm not so sure if multiselectable is needed for the classic treeBehavior. Because we decided to go with `aria-checkbox` there

Reply:
Hi @assuncaocharles & @YuanboXue-Amber agree with both of you :)
based on aria: 
https://www.w3.org/TR/wai-aria-1.1/#aria-multiselectable
looks like it is related only to the 'aria-selected'. So until we have 'aria-selected' it would be good to add. 
But when we are going to change it into aria-checkbox (high probably), then we wouldn't add.

Reply:
You're right, it should be there for normal tree behaviour as well. I just did a small test on multiselect tree with normal behaviour, the accessibility computed looks like:
```
Name: "default"
aria-labelledby: Not specified
aria-label: "default" 
title: Not specified
Role: tree
Focusable: true
Multi-selectable: false
Required: false
```
And `Multi-selectable: false` will turn `true` if I added 'aria-multiselectable'

Surprisingly NVDA didn't narrate differently with/without 'aria-multiselectable'. But I still think we should put it there.

Reply:
Resolved by [74a6595](https://github.com/microsoft/fluentui/pull/15147/commits/74a6595154110b452669287ef28cf16e4ba108db)